sdk/src/main/java/com/mapbox/maps/renderer/widget/BitmapWidget.kt

```diff
@@ -20,16 +21,20 @@ open class BitmapWidget @JvmOverloads constructor(
   ),
   marginX: Float = 0f,
   marginY: Float = 0f,
+  onBitmapLoaded: (Bitmap) -> Unit = {},
```

Comment:
Instead of lambda (which is seen as `FunctionN` in Java) should we use a proper [SAM](https://kotlinlang.org/docs/fun-interfaces.html)?

Reply:
But, if you do so, don't we lose the nice benefit of being able to write:
```Kotlin
// Creating an instance using lambda
val isEven = IntPredicate { it % 2 == 0 }
```
instead of:
```Kotlin
// Creating an instance of a class
val isEven = object : IntPredicate {
   override fun accept(i: Int): Boolean {
       return i % 2 == 0
   }
}
```

Reply:
Hmmm `fun interface` does provide a nicer syntax for the caller. We'll use that for now at least.